services/app-api/src/resolvers/questionResponse/createQuestion.ts

```diff
@@ -104,8 +104,18 @@ export function createQuestionResolver(
             setErrorAttributesOnActiveSpan(errMessage, span)
             throw new Error(errMessage)
         }
-
-        const questionResult = await store.insertQuestion(input, user)
+        const docs = input.documents.map((doc) => {
+            return {
+                name: doc.name,
+                s3URL: doc.s3URL,
+                downloadURL: doc.downloadURL ?? undefined,
```

Comment:
How come we have downloadURL in the document type here? I thought we would only add it to the graphql type, added at the last second with the resolver. It should definitely not end up in the db (haven't gotten that far in the review yet) because it's only temporary, right?